src/olympia/amo/admin.py

```diff
@@ -521,7 +521,7 @@ def known_ip_adresses(self, obj):
             )
 
         contents = format_html_join(
-            '', '<li>{}</li>', ((ip,) for ip in sorted(ip_addresses))
+            '', '<li>{}</li>', sorted((str(i),) for i in ip_addresses)
```

Comment:
It'd be ideal if any ipv6 addresses came after all the ipv4 addresses really... but not failing is the priority so 🤷‍♂️